recipes/cesium/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "cesium" %}
+{% set version = "0.9.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7cbde6817c00e1f0c3e51aaf860cf7784dfc58766f2d4e68a6ca613be10d3d81"
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - numpy >=1.14.3
+    - cython
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - numpy >=1.14.3
```

Comment:
```suggestion
```
 You can drop this second numpy line completely, because `{{ pin_compatible('numpy') }}` will fill in the required version you gave in the `host` section.